siriuspy/siriuspy/sofb/main.py

```diff
@@ -840,6 +851,11 @@ def _do_auto_corr(self):
                 self._update_log(msg)
                 _log.info(msg)
                 break
+            if not self.is_amc_locked:
+                msg = 'ERR: Cannot Correct, at least one AMC is not locked!'
+                self._update_log(msg)
+                _log.info(msg)
+                break
```

Comment:
When we had connection issues, this error message was somewhat misleading, because it suggested that the timing wasn't locked, when in fact the issue was that the PV wasn't connected.

Do you think it would be easy to differentiate between the two issues in the log message?

If we lost connection to one of the crates, it still makes sense to open the SOFB loop, so the overall behavior should remain, I think

Reply:
Right, done!